content/en/docs/security/otel-collector-configuration-best-practices.md

```diff
@@ -0,0 +1,147 @@
+definition.
+
+### Encryption and Authentication
+
+<!-- TODO: SHOULD use encryption and authentication. How do you configure that?
+
+  **NOTE**: MAY pose a security risk if configuration parameters are modified improperly -->
+
+## Processors
+
+Processors sit between receivers and exporters. They are responsible for
+processing the data in some way. From a security perspective, they are useful in
+a few ways.
+
+### Recommended Processors
+
+<!--- TODO: SHOULD configure recommended processors. If so, what are they? -->
+
+### Scrubbing sensitive data
+
+It is common for an OpenTelemetry Collector to be used to scrub sensitive data
+before exporting it to a backend. This is especially important when sending the
+data to a third-party backend. The OpenTelemetry Collector **SHOULD** be
+configured to obfuscate or scrub sensitive data before exporting.
+
+<!--- TODO: SHOULD configure obfuscation/scrubbing of sensitive metadata. How? Give more details and/or link to an existing document -->
+
+Use OpenTelemetry Collector's `redaction` processor to scrub sensitive data.
+
+### Safeguarding around resource utilization
+
+Processors also offer safeguards around resource utilization. After implementing
+[safeguards around resource utilization in your hosting](/security/otel-collector-hosting-best-practices/),
+ensure your OpenTelemetry Collector configuration uses these safeguards.
+
+<!-- start same page content in otel-collector-hosting-best-practices -->
+
+The `batch` and especially `memory_limiter` processor help ensure that the
+OpenTelemetry Collector is resource efficient and does not out of memory when
+overloaded. At least these two processors SHOULD be enabled on every defined
+pipeline.
+
+For more information on recommended processors and order, refer to the
+[OpenTelemetry Collector Processor](https://github.com/open-telemetry/opentelemetry-collector/tree/main/processor)
+documentation.
+
+<!-- /end same page content in otel-collector-hosting-best-practices -->
+
+## Extensions
+
+While receivers, processors, and exporters handle telemetry data directly,
+extensions typical serve different needs.
+
+<!--- TODO: Extensions SHOULD NOT expose sensitive health or telemetry data. How? What can you do? -->
+
+### Health and Telemetry
+
+The initial extensions provided health check information, Collector metrics and
+traces, and the ability to generate and collect profiling data. When enabled
+with their default settings, all of these extensions except the health check
+extension are only accessibly locally to the OpenTelemetry Collector. Care
+should be taken when configuring these extensions for remote access as sensitive
+information may be exposed as a result.
+
+### Collector's Internal Telemetry
+
+<!--- INSERT RECOMMENDATIONS HERE. For example:
+
+1. Remove zPages.
+1. Remove configuration endpoints.
+-->
+
+### Forwarding [component developers?]
```

Comment:
Am not familiar with Forwarding. Is this content related to component developers? If so, I can remove it from this PR for end users.

Reply:
I would keep this out, at least for now.